docs/tutorials/etl.md

```diff
@@ -0,0 +1,240 @@
+and 3) We will exercise the ingested data using a simple project.
+
+It is assumed throughout this tutorial that Spark 2.0.0 or greater is installed,
+that the GDAL command line tools are installed,
+and that the GeoTrellis source tree has been locally cloned.
+
+# Local ETL #
+
+## Build the Assembly ##
+
+Navigate into the GeoTrellis source tree, build the assembly, and copy it to some convenient location:
+
+```console
+cd geotrellis
+./sbt "project spark-etl" assembly
+cp spark-etl/target/scala-2.11/geotrellis-spark-etl-assembly-1.0.0.jar /tmp
+```
+
+Although in this tutorial we have chosen to build this assembly directly from the GeoTrellis source tree,
+in some applications it may be desirable to create a class in one's own code base that uses `geotrellis.spark.etl.SinglebandIngest` or `geotrellis.spark.etl.MultibandIngest` and generate an assembly of that.
+See the [Chatta Demo](https://github.com/geotrellis/geotrellis-chatta-demo/blob/94ae99269236610e66841893990860b7760e3663/geotrellis/src/main/scala/geotrellis/chatta/ChattaIngest.scala) for an example.
+
+## Compose JSON Configuration Files ##
+
+The configuration files that we create in this section are intended for use with a single multiband GeoTiff image.
+Three JSON files are required: one describing the input data, one describing the output data, and one describing the backend(s) in which the catalog should be stored.
+Please see [the ETL documentation](../spark-etl/spark-etl-run-examples.md) for more information on this topic.
+
+First, retile the source raster.
+This is not strictly required if the source image is small enough (probably less than 2GB),
+but is still good practice even if it is not required.
+
+```console
+gdal_retile.py source.tif -of GTiff -co compress=deflate -ps 256 256 -targetDir /tmp/rasters
+```
+
+The result of this command is a collection of smaller GeoTiff tiles in the directory `/tmp/rasters`.
+
+We will now create three files in the `/tmp/json` directory: `input.json`, `output.json`, and `backend-profiles.json`.
+
+Here is `input.json`:
+```json
+[{
+    "format": "multiband-geotiff",
+    "name": "example",
+    "cache": "NONE",
+    "backend": {
+        "type": "hadoop",
+        "path": "file:///tmp/rasters"
+    }
+}]
+```
+
+The value `multiband-geotiff` is associated with the `format` key.
+That is required if you want to access the data as an RDD of key, `MultibandTile` pairs.
+Making that value `geotiff` instead of `multiband-geotiff` will result in `Tile` data.
+The value `example` associated with the key `name` gives the name of the layer(s) that will be created.
+The `cache` key gives the Spark caching strategy that will be used during the ETL process.
+Finally, the value associated with the `backend` key specifies where the data should be read from.
+In this case, the source data are stored in the directory `/tmp/rasters` on local filesystem and accessed via Hadoop.
```

Comment:
Do we have a documented grammar of the ETL json fields/options? If so, it would be handy to incorporate it here.